frontend/packages/dev-console/src/components/NamespacedPage.tsx

```diff
@@ -23,10 +24,14 @@ const NamespacedPage: React.FC<NamespacedPageProps> = ({
   onNamespaceChange,
   hideApplications = false,
   variant = NamespacedPageVariants.default,
+  toolbarOptions = undefined,
 }) => (
   <div className="odc-namespaced-page">
     <NamespaceBar disabled={disabled} onNamespaceChange={onNamespaceChange}>
       {!hideApplications && <ApplicationSelector disabled={disabled} />}
+      {toolbarOptions && (
+        <div className="odc-namespaced-page__toolbar-options">{toolbarOptions}</div>
+      )}
```

Comment:
This is not the best way to do this... this gets mounted under a component for dropdowns.
![image](https://user-images.githubusercontent.com/8126518/67396223-2e672080-f575-11e9-8337-aebd2efd491d.png)

Recommend adding "actionItems" (feel free to change the name) to the `NamespaceBar` component:
```
const NamespaceBar_ = ({ useProjects, children, disabled, onNamespaceChange, actionItems }) => {
  return (
    <div className="co-namespace-bar" style={{ display: 'flex', alignItems: 'center' }}>
      <div style={{ flex: '1 1 auto' }}>
        <Firehose resources={[{ kind: getModel(useProjects).kind, prop: 'namespace', isList: true }]}>
          <NamespaceBarDropdowns
            useProjects={useProjects}
            disabled={disabled}
            onNamespaceChange={onNamespaceChange}
          >
            {children}
          </NamespaceBarDropdowns>
        </Firehose>
      </div>
      {actionItems && (
        <div style={{ flex: '0 0 auto' }}>
          {actionItems}
        </div>
      )}
    </div>
  );
};
```
Obviously don't use inline styles, they are just showcasing what I did.

Just pass in your `toolbarOptions` for it to mount them in a place the NamespaceBar desires:
```
    <NamespaceBar disabled={disabled} onNamespaceChange={onNamespaceChange} actionItems={toolbarOptions}>
      {!hideApplications && <ApplicationSelector disabled={disabled} />}
    </NamespaceBar>
```

Reply:
Furthermore, as suggested in the first review, the toolbar placement should be owned by the `NamespaceBar`. Use a `prop` on the `NamespaceBar` to pass in the buttons you need and let it place it in the far right of the bar.

Reply:
You still need to address this @vikram-raj - we can't be mounting a toolbar item under the "NamespaceBarDropdowns"